pkgs/os-specific/linux/xmm7360-pci/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch, kernel, perl, bc, breakpointHook }:
+
+stdenv.mkDerivation rec {
+  pname = "xmm7360-pci";
+  version = "unstable-2021-05-18";
+
+  src = fetchFromGitHub {
+    owner = "xmm7360";
+    repo = "xmm7360-pci";
+    rev = "9d9a2000cc70fc941e771e35f579becd9afe7445";
+    sha256 = "158m9sgz59dqsklkzrll125j7bi63kwbqxvn214dl5hnnb8haxc8";
+  };
+
+  # https://github.com/xmm7360/xmm7360-pci/pull/117
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/xmm7360/xmm7360-pci/pull/117/commits/1379b07afb093d56ec63ae9d8d6f02876e0bcd81.patch";
+      sha256 = "1mcdbbckda0s402s7m33mp240gi7899jca0rl7kinwryaj7pk85r";
+    })
+  ];
+
+  makeFlags = [ "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build" ];
+
+  nativeBuildInputs = [ perl bc ] ++ kernel.moduleBuildDependencies;
```

Comment:
I'm not convinced that perl and bc are required for this build.

Reply:
True. I removed it.